### PR TITLE
Update the resource pages

### PR DESCRIPTION
**Pull request by tas50**
Improve the markdown formatting
Don't document deprecated properties

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: content/resources/yum_repository/_index.md [hunk 1]
```diff
@@ -21,135 +47,126 @@ resource_description_list:
...
-registry_key: false
-nameless_apt_update: false
-nameless_build_essential: false
-resource_package_options: false
+syntax_full_properties_list:
+- "`yum_repository` is the resource."
+- "`name` is the name given to the resource block."
+- "`action` identifies which steps Chef Infra Client will take to bring the node into
+  the desired state."
+- "``baseurl``, , ``clean_metadata``, ``cost``, ``description``, ``enabled``, ``enablegroups``,
```
Comment (kagarmoe) [inline content/resources/yum_repository/_index.md]:
```suggestion
- "``baseurl``, ``clean_metadata``, ``cost``, ``description``, ``enabled``, ``enablegroups``,
```
You're generating an extra comma.

File: content/resources/yum_repository/_index.md [hunk 2]
```diff
@@ -21,135 +47,126 @@ resource_description_list:
...
   required: false
-  default_value: null
-  new_in: null
+  default_value:
+  new_in:
   description_list:
-  - markdown: 'Relative cost of accessing this repository. Useful for weighing one
-
-      repo''s packages as greater/less than any other.'
+  - markdown: Relative cost of accessing this repository. Useful for weighing one
```
Comment (kagarmoe) [inline content/resources/yum_repository/_index.md:137]: Do you mean cost in terms of $ or by some other metric (time, etc.)?

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: content/resources/yum_repository/_index.md (hunk 1 above)
Comment (tas50) [reply content/resources/yum_repository/_index.md]: Let me see what in the automation caused that one

File: content/resources/yum_repository/_index.md (hunk 2 above)
Comment (tas50) [reply content/resources/yum_repository/_index.md:137]: Computery costs? 🤷

File: content/resources/yum_repository/_index.md (hunk 1 above)
Comment (tas50) [reply content/resources/yum_repository/_index.md]: Turns out the automation got it right and I missed it. That's commited now.

**Review by phiggins — APPROVED** [review]

File: content/resources/yum_repository/_index.md (hunk 2 above)
Comment (phiggins) [reply content/resources/yum_repository/_index.md:137]: FWIW this is pretty much word for word copied from the [official description](https://linux.die.net/man/5/yum.conf): 

> cost relative cost of accessing this repository. Useful for weighing one repo's packages as greater/less than any other. defaults to 1000 

[Ansible uses the same description](https://docs.ansible.com/ansible/latest/modules/yum_repository_module.html).

Seems like an awkward way of describing priority?

Comment (kagarmoe) [reply content/resources/yum_repository/_index.md:137]: At least we're not alone in our vagueness.

**Review by kagarmoe — APPROVED** [review]
tenor-171728502

**Pull request merged**